decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -161,20 +139,34 @@ def current_user_unread_data
       end
     end
 
-    def organization_description_label
-      @organization_description_label ||= if empty_organization_description?
-                                            t("decidim.pages.home.footer_sub_hero.footer_sub_hero_body_html")
-                                          else
-                                            decidim_sanitize_admin(translated_attribute(current_organization.description))
-                                          end
+    def current_url(params = request.parameters)
+      return url_for(params) if respond_to?(:current_participatory_space) || respond_to?(:current_component)
+
+      each_decidim_engine do |helpers|
+        return helpers.url_for(params)
+      rescue ActionController::UrlGenerationError
+        # Continue to next engine in case the URL is not available.
+      end
+
+      main_app.url_for(params)
+    rescue ActionController::UrlGenerationError
+      "#{request.base_url}#{"?#{params.to_query}" unless params.empty?}"
     end
 
     private
 
-    def empty_organization_description?
```

Comment:
@andreslucena , i believe removing this method is outside the scope of this backport. Could you revert?